packages/owl-bot/src/handlers.ts

```diff
@@ -257,69 +252,37 @@ export async function refreshConfigs(
     commitHash: commitHash,
   };
 
-  // Query github for the contents of the lock file.
-  const lockContent = await core.getFileContent(
-    githubOrg,
-    repoName,
-    owlBotLockPath,
-    commitHash,
-    octokit
-  );
-  if (lockContent) {
-    try {
-      newConfigs.lock = owlBotLockFrom(
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        yaml.load(lockContent) as Record<string, any>
-      );
-    } catch (e) {
-      logger.error(
-        `${repoFull} has an invalid ${owlBotLockPath} file: ${e.message}`
-      );
-
-      const title = `Invalid ${owlBotLockPath}`;
-      const body = `\`owl-bot\` will not be able to update this repo until '${owlBotLockPath}' is fixed.
-
-Please fix this as soon as possible so that your repository will not go stale.`;
-
-      await createIssueIfTitleDoesntExist(
-        octokit,
-        githubOrg,
-        repoName,
-        title,
-        body,
-        logger
-      );
+  try {
+    const [lock, yamls] = await fetchConfigs(octokit, {
```

Comment:
This will make it easier for us to support config files regardless of case sensitivity or `yaml` vs., `yml` suffix, see:

https://github.com/googleapis/repo-automation-bots/issues/1573